src/main/java/com/stuypulse/robot/commands/ArcadeDrive.java

```diff
@@ -0,0 +1,45 @@
+package com.stuypulse.robot.commands;
+
+import static com.stuypulse.robot.constants.Settings.Driver.*;
+import static com.stuypulse.robot.constants.Settings.Edwin.Motion.*;
+
+import com.stuypulse.robot.subsystems.Robot;
+import com.stuypulse.stuylib.input.Gamepad;
+import com.stuypulse.stuylib.math.SLMath;
+import com.stuypulse.stuylib.streams.IStream;
+import com.stuypulse.stuylib.streams.filters.LowPassFilter;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ArcadeDrive extends CommandBase {
+
+    private final Robot robot;
+    private final IStream speed, angle;
+
+    public ArcadeDrive(Robot robot, Gamepad gamepad) {
+        this.robot = robot;
+
+        speed = IStream.create(() -> gamepad.getRightTrigger() - gamepad.getLeftTrigger()).filtered(
+                                x -> SLMath.deadband(x, SPEED_DEADBAND.get()),
+                                x -> SLMath.spow(x, SPEED_POWER.get()),
+                                x -> x * MAX_VELOCITY,
+                                new LowPassFilter(SPEED_FILTER));
+
+        angle = IStream.create(() -> gamepad.getLeftX()).filtered(
+                                x -> SLMath.deadband(x, ANGLE_DEADBAND.get()),
```

Comment:
these max velocities change per .. make a custom `Constraints` class that just stores max velocity, max accel, trackwidth, and a reference to the kinematics object. have a toTrajectoryConstraint method. 

this will allow for u to access the maxVelocity and maxAngularVelocity in any command AND allow u to pass in trajectoryConstraints when needed

there's probably more to this subject but just creating a structure for storing all the values so that we can get them back whenever we want is good enough for now